libfuse/constants_darwin.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
```

Comment:
if you name a file `_darwin.go` the build line is implied..
http://blog.ralch.com/tutorial/golang-conditional-compilation/#file-suffixes:0765dbe650f7259dfb0634d2c3467e31